src/pages/overview/pdf-services-api/howtos/reorder-pages.md

```diff
@@ -32,44 +32,54 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
-                    .withClientId("PDF_SERVICES_CLIENT_ID")
-                    .withClientSecret("PDF_SERVICES_CLIENT_SECRET")
-                    .build();
+                   // Initial setup, create credentials instance
+            Credentials credentials = new ServicePrincipalCredentials(
+                    System.getenv("PDF_SERVICES_CLIENT_ID"),
+                    System.getenv("PDF_SERVICES_CLIENT_SECRET"));
+
+            // Creates a PDF Services instance
+            PDFServices pdfServices = new PDFServices(credentials);
+
+            // Creates an asset from source file and upload
+            InputStream inputStream = Files.newInputStream(new File("src/main/resources/reorderPagesInput.pdf").toPath());
+            Asset asset = pdfServices.upload(inputStream, PDFServicesMediaType.PDF.getMediaType());
 
-           // Create an ExecutionContext using credentials and create a new operation instance.
-           ExecutionContext executionContext = ExecutionContext.create(credentials);
-           ReorderPagesOperation reorderPagesOperation = ReorderPagesOperation.createNew();
+            PageRanges pagesToReorder = getPageRangeForReorder();
+
+            // Create parameters for the job
+            ReorderPagesParams reorderPagesParams = ReorderPagesParams
+                    .reorderPagesParamsBuilder(asset, pagesToReorder) // Add the asset as input to the params, along with its page order
+                    .build();
 
-           // Set operation input from a source file, along with specifying the order of the pages for
-           // rearranging the pages in a PDF file.
-           FileRef source = FileRef.createFromLocalFile("src/main/resources/reorderPagesInput.pdf");
-           PageRanges pageRanges = getPageRangeForReorder();
-           reorderPagesOperation.setInput(source);
-           reorderPagesOperation.setPagesOrder(pageRanges);
+            // Creates a new job instance
+            ReorderPagesPDFJob reorderPagesPDFJob = new ReorderPagesPDFJob(reorderPagesParams);
 
-           // Execute the operation.
-           FileRef result = reorderPagesOperation.execute(executionContext);
+            // Submits the job and gets the job result
+            String location = pdfServices.submit(reorderPagesPDFJob);
+            PDFServicesResponse<ReorderPagesResult> pdfServicesResponse = pdfServices.getJobResult(location, ReorderPagesResult.class);
 
-           // Save the result to the specified location.
-           result.saveAs("output/reorderPagesOutput.pdf");
+            // Get content from the resulting asset(s)
+            Asset resultAsset = pdfServicesResponse.getResult().getAsset();
+            StreamAsset streamAsset = pdfServices.getContent(resultAsset);
 
-       } catch (IOException | ServiceApiException | SdkException | ServiceUsageException e) {
+            // Creates an output stream and copy stream asset's content to it
```

Comment:
creates -- > copies

Reply:
As discussed, IG this is from already reviewed samples.